Aquality.Selenium.Template/Aquality.Selenium.Template.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29102.190
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Aquality.Selenium.Template.SpecFlow", "Aquality.Selenium.Template.SpecFlow\Aquality.Selenium.Template.SpecFlow.csproj", "{21B3F61F-E039-492C-A959-73049136EB4A}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Aquality.Selenium.Template", "Aquality.Selenium.Template\Aquality.Selenium.Template.csproj", "{84ADF59F-0C6C-4C35-BB35-D393571939D2}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{21B3F61F-E039-492C-A959-73049136EB4A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{21B3F61F-E039-492C-A959-73049136EB4A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{21B3F61F-E039-492C-A959-73049136EB4A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{21B3F61F-E039-492C-A959-73049136EB4A}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
wonder do we really need both "Debug" and "Release" configurations for all projects in autotest solution

Reply:
fixed